source/frontend/src/features/leases/add/AddLeaseContainer.tsx

```diff
@@ -52,9 +52,10 @@ export const AddLeaseContainer: React.FunctionComponent<
     formikHelpers: FormikHelpers<LeaseFormModel>,
     userOverrideCodes: UserOverrideCode[],
   ) => {
-    const leaseApi = leaseFormModel.toApi();
+    const leaseApi = LeaseFormModel.toApi(leaseFormModel);
```

Comment:
q: given that the leaseFormModel has all the information for the transformation to the API version calling it as static is not necessary. What advantage do you see in it being static?

Reply:
There are some problems with non-static functions in classes that we were previously working around with any or as,
primarily it prevents some usages of ellipsis (...) which can be an issue for default* objects as well as test methods.

using static methods allows typescript to treat these models as if they were simple objects which makes transformations easier.